src/main/target/OMNIBUSF7/target.h

```diff
@@ -168,6 +171,7 @@
 #define DEFAULT_RX_FEATURE      FEATURE_RX_SERIAL
 #define SERIALRX_UART           SERIAL_PORT_USART2
 #define SERIALRX_PROVIDER       SERIALRX_SBUS
+#define ESC_SENSOR_UART         SERIAL_PORT_USART7
```

Comment:
Actually, I was wondering how something like this could be handled in 3.3. If we move them to config.c, we would have to generate targets with different `config.c`. Thoughts?